src/lib/Runner.ts

```diff
@@ -556,17 +567,6 @@ export class Runner {
     return statuses;
   };
 
-  getLandRequestQueuedDate = async (requestId: string): Promise<Date | null> => {
-    const status = await LandRequestStatus.findOne<LandRequestStatus>({
-      where: {
-        requestId,
-        state: 'queued',
-      },
-      order: [['date', 'ASC']],
-    });
-    return status ? status.date : null;
-  };
```

Comment:
I just realised that I was wrong about not needing this function anymore, will need to replace `landRequestStatus.date.getTime()` with what we had before, sorry about that.